tendrl/notifier/utils/central_store_util.py

```diff
@@ -0,0 +1,57 @@
+from etcd import EtcdException
+from etcd import EtcdKeyNotFound
+from tendrl.commons.objects.alert import Alert
+from tendrl.commons.objects.cluster_alert import ClusterAlert
+from tendrl.commons.objects.node_alert import NodeAlert
+from tendrl.commons.utils import etcd_utils
+
+
+def read(key):
+    result = {}
+    job = {}
+    job = etcd_utils.read(key)
+    if hasattr(job, 'leaves'):
+        for item in job.leaves:
+            if key == item.key:
+                result[item.key.split("/")[-1]] = item.value
+                return result
+            if item.dir is True:
+                result[item.key.split("/")[-1]] = read(item.key)
+            else:
+                result[item.key.split("/")[-1]] = item.value
+    return result
+
+
+def get_alerts():
+    alerts_arr = []
+    alerts = read('/alerting/alerts')
+    for alert_id, alert in alerts.iteritems():
+        alerts_arr.append(Alert(**alert))
+    return alerts_arr
+
+
+def update_alert_delivery(alert):
+    alert.delivery = "True"
+    # update alert
+    alert.save()
+    if "alert_catagory" in alert.tags:
+        if alert.tags['alert_catagory'] == 'cluster':
```

Comment:
based on alert category only read key is changing so the code could be something like 

```
if alert.tags['alert_category'] == 'cluster':
    key = '/alerting/clusters/%s/%s' % (alert.tags['integration_id'], alert.alert_id)
elif alert.tags['alert_category'] == 'node':
    key = '/alerting/nodes/%s/%s' % (alert.node_id, alert.alert_id)

alert_obj = read(key)
alert_obj['delivery'] = alert.delivery
NodeAlert(**alert_obj).save()
```